### PR TITLE
Feature/progress

### PR DESCRIPTION
**Pull request by iiskaandar**

### CODE REVIEW

File: components/Progress/LineProgress/styles.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  strokeLinecap?: stroke;
+  strokeHeight?: string;
+}
+
+export const Tracker = styled.div`
+  width: ${(props: TrackerProps) => props.width + '%'};
+  height: ${(props: TrackerProps) =>
+    props.strokeHeight ? `${props.strokeHeight}px` : '10px'};
+  margin-right: 10px;
+  background: #ffffff;
```
Comment (adamkosmala) [inline components/Progress/LineProgress/styles.ts]: please do not use shorthand syntax, use `background-color` instead

File: components/Progress/LineProgress/Progress.story.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    readme: {
+      sidebar: Readme,
+    },
+  })
+  .add('Line progress', () => {
+    return (
+      <LineProgress
+        value={number('Value', 50)}
+        showLabel={boolean('Show label', true)}
+        width={number('Width', 50)}
```
[the remark this answers is not in the document]
Comment (iiskaandar) [reply components/Progress/LineProgress/Progress.story.tsx]: done

File: components/Progress/LineProgress/Progress.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+import { Tracker, ProgressInTracker, ProgressContainer } from './styles';
+import { stroke } from '../types';
+
+interface ProgressProps {
+  value: number;
+  showLabel?: boolean;
+  width: number;
+  color: string;
+  format?: string;
+  strokeLinecap?: stroke;
```
[the remark this answers is not in the document]
Comment (iiskaandar) [reply components/Progress/LineProgress/Progress.tsx]: done

File: components/Progress/LineProgress/styles.ts (hunk 1 above)
Comment (iiskaandar) [reply components/Progress/LineProgress/styles.ts]: Done

**Pull request merged**